platform/nephos/nephos-modules/modules/init.d/nps-modules-4.9.0-9-2-amd64

```diff
@@ -25,7 +25,8 @@ start)
             echo "25165824" > /proc/sys/net/core/wmem_max
         fi
 
-        modprobe nps_dev
+	depmod -a
```

Comment:
depmod should not be needed. the depmod is only needed at the debian package installation time.